Tests/SwiftKueryORMTests/TestUpdate.swift

```diff
@@ -0,0 +1,51 @@
+import XCTest
+
+@testable import SwiftKueryORM
+import Foundation
+import KituraContracts
+
+class TestUpdate: XCTestCase {
+    static var allTests: [(String, (TestUpdate) -> () throws -> Void)] {
+        return [
+            ("testUpdate", testUpdate),
+        ]
+    }
+
+    struct Person: Model {
+        static var tableName = "People"
+        var name: String
+        var age: Int
+    }
+
+    /**
+      Testing that the correct SQL Query is created to update a specific model.
+    */
+    func testUpdate() {
+        let connection: TestConnection = createConnection()
+        Database.default = Database(single: connection)
+        performTest(asyncTasks: { expectation in
+            let person = Person(name: "Joe", age: 38)
+            person.update(id: 1) { p, error in
+                XCTAssertNil(error, "Update Failed: \(String(describing: error))")
+                XCTAssertNotNil(connection.query, "Update Failed: Query is nil")
+                if let query = connection.query {
+                  let expectedPrefix = "UPDATE People SET"
+                  let expectedSuffix = "WHERE People.id = '1'"
+                  let expectedUpdates = ["name = 'Joe'", "age = 38"]
+                  let resultQuery = connection.descriptionOf(query: query)
+                  XCTAssertTrue(resultQuery.hasPrefix(expectedPrefix))
+                  XCTAssertTrue(resultQuery.hasSuffix(expectedSuffix))
+                  for update in expectedUpdates {
+                      XCTAssertTrue(resultQuery.contains(update))
+                  }
```

Comment:
This looks good, would it be worth a line comment to explain why we are doing it this way? (ie. that the ordering of the update statements is unpredictable due to the data structure used to store them within the ORM)